rust/telemetry/src/lib.rs

```diff
@@ -51,13 +51,16 @@ impl Telemetry {
         tracing::info!("Starting telemetry");
         let inner = sentry::init((
             dsn.0,
+            // Can't use URLs as `environment` here, because Sentry doesn't allow slashes in environments.
```

Comment:
Should we match on our well-known environments and fallback to something else if the customer is self-hosted?

- `api.firezone.dev` => `production`
- `api.firez.one` => `staging`
- other => `self-hosted`